va/validation-authority.go

```diff
@@ -259,44 +254,35 @@ func (va *ValidationAuthorityImpl) fetchHTTP(identifier core.AcmeIdentifier, pat
 	}
 	httpResponse, err := client.Do(httpRequest)
 	if err != nil {
-		challenge.Status = core.StatusInvalid
-		challenge.Error = &probs.ProblemDetails{
+		va.log.Debug(err.Error())
```

Comment:
I agree it's a weird debug log. I was just moving the existing one below and cleaning it up a bit. There are a bunch of similar ones in this file, filed an issue to fix: https://github.com/letsencrypt/boulder/issues/1281